src/Serializer/BaseSerializerHandler.php

```diff
@@ -75,13 +77,22 @@ public static function getSubscribingMethods(): array
     /**
      * Serialize data object to id.
      *
+     * NEXT_MAJOR: Update typehint when dropping jms/serializer < 3.0
```

Comment:
IMO, we should specify at least which argument must be updated.
```suggestion
     * NEXT_MAJOR: Update type hint when dropping jms/serializer < 3.0
```